.github/workflows/csti.yml

```diff
@@ -34,16 +34,20 @@ jobs:
     runs-on: ubuntu-latest
     needs: setup
     steps:
+      - name: Use GitHub App Token
+        uses: wow-actions/use-app-token@d7957e08172ca2e8e49b35b8d266ad585885edc7
```

Comment:
why are we using this commit hash rather than [v2.0.2](https://github.com/wow-actions/use-app-token/tree/v2.0.2), the version that's attached to it?

Reply:
A couple of reasons:
- This is a third party action, and it's producing an API token, which feels scary, so following [the guidance](https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions).
- It's what https://github.com/github/combine-prs/blob/main/docs/github-app-setup.md does.

Reply:
gotcha. in that case i've added a suggestion for a comment to make the reasoning a bit clearer